examples/va/nvidia-mdev/edpm/nodeset/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../../../va/nvidia-mdev/edpm/nodeset
```

Comment:
Needs to remove a parent directory: `../../../../../va/nvidia-mdev/edpm/nodeset`